Groovy/Groovy.sublime-syntax

```diff
@@ -744,21 +751,21 @@ contexts:
       scope: punctuation.definition.separator.groovy
 
   primitive-types:
-    - match: \bbyte(?:\b|(?:\[\s*\])*\b)
+    - match: \bbyte(?:\b|{{optional_arrays}}\b)
       scope: storage.type.primitive.integer.8bit.groovy
-    - match: \bshort(?:\b|(?:\[\s*\])*\b)
+    - match: \bshort(?:\b|{{optional_arrays}}\b)
       scope: storage.type.primitive.integer.16bit.groovy
-    - match: \bint(?:\b|(?:\[\s*\])*\b)
+    - match: \bint(?:\b|{{optional_arrays}}\b)
       scope: storage.type.primitive.integer.32bit.groovy
-    - match: \blong(?:\b|(?:\[\s*\])*\b)
+    - match: \blong(?:\b|{{optional_arrays}}\b)
       scope: storage.type.primitive.integer.64bit.groovy
-    - match: \bfloat(?:\b|(?:\[\s*\])*\b)
+    - match: \bfloat(?:\b|{{optional_arrays}}\b)
       scope: storage.type.primitive.float.32bit.groovy
-    - match: \bdouble(?:\b|(?:\[\s*\])*\b)
+    - match: \bdouble(?:\b|{{optional_arrays}}\b)
```

Comment:
Why do the optional arrays have `\b` at the end? Won't that basically never match?

Reply:
Right. I know this wasn't your doing. Just wanted to flag. 🙂